docs/Call_for_papers_Bioinformatics.md

```diff
@@ -0,0 +1,132 @@
+
+*   Sensores Bioinspirados
+*   Bio-impresión
+*   Bioplasticos
+*   Simulaciones y Procesamiento del Cerebro, Corazón y otros órganos
+*   Lecturas cerebrales y Neuroinformática
+*   Interfaces Cerebro-Máquina
+*   Cancer and Tumor Studies with Big Data
+*   Conductive polymers
+*   Secuenciación de ADN
+*   Informática para la Salud como Servicio (HIaaS)
+*   Inmunoensayos
+*   In vitro meat
+*   Lab-on-a-chip y Aplicaciones
+*   Nanomedicinas y Nanosensores
+*   Oncolytic virus research
+*   Tratamientos Personalizados
+*   Plantibodies
+*   Stem cell treatments
+*   Lenguajes Abierto para Biología Sintética.
+*   Ingeniería de Tejidos
+
+En general el comité del evento aceptará artículos relacionados con las siguientes disciplinas:
+
+        Analytical Chemistry, Astrobiology, Biochemistry, Bioinformatics,
+        Biomaterials,    Biomechatronics, Biophysics, Biotechnology,
+        Chemical Engineering, Chemical Biology, Cheminformatics, Computational Biology,
+        Computational Chemistry, Femtochemistry, Flow Chemistry, Genetics, Genomics, 
+        Healthcare Science, Immunochemistry, Immunology, Mathematical Biology,
+        Mathematical Chemistry, Microbiology, Molecular Biology, Molecular Mechanics,
+        Nanomaterials, Nanotechnology, Neuroscience, Paleobiology,
+        Polymer Chemistry and Engineering, Phytochemistry, Structural Biology, 
+        Supramolecular Chemistry, Synthetic Biology, Synthetic Genomics Systematics, 
+        Tissue Engineering, Virology.
+
+## Otros Talleres de SciPyLA 2017
+
+*   Física
+*   Ingeniería Eléctrica y Automática
+*   Economía e Investigación de Operaciones
+*   Tecnologías Emergentes de Cómputo
+*   Ciencias de la Tierra
+*   Ciencias Sociales y Humanidades
+
+## Actividades Previstas
+
+* **Tutoriales** : Talleres ofrecidos por expertos dirigidas a mejorar la productividad de los investigadores.
+
+* **Presentaciones y charlas:** Conferencistas invitados hablarán acerca de tendencias mundiales y mejores prácticas en el ámbito científico.
+
+*   **Sesiones de Posters:** Exponga sus investigaciones utilizando pancartas y medios visuales atractivos.
+
+*   **Sprints:** Reúnase con los autores de bibliotecas singulares para conocer maneras más eficientes de realizar cálculos científicos.
+
+*   **Charlas relámpago:** Promocione y amplíe el alcance de su investigación exponiendo en breve tiempo sus ideas a los especialistas de la región.
+
+# Subscripción / Asistencia / Pagos
+
+El evento tendrá subscripción libre. Su participación dependerá del envío de trabajos y la calidad del mismo. Sí es seleccionado usted deberá garantizar su estancia en La Habana a través de paquetes turísticos no ofrecidos directamente por la conferencia.
+
+* Para facilitar su experiencia en un viaje a Cuba le recomendamos leer [Instrucciones para viajar a Cuba](http://scipyla.org/conf/2017/cubatravelinstructions.html)
+* [Paquetes turísticos ofrecidos por PYMES en Cuba](http://cubawalkingtour.com)
```

Comment:
Link should be http://scipyla.org/conf/2017/travel/tours and caption should be *Paquetes turísticos recomendados por la conferencia*.